src/components/structures/LoggedInView.tsx

```diff
@@ -531,6 +533,33 @@ class LoggedInView extends React.Component<IProps, IState> {
                 // if we do not have a handler for it, pass it to the platform which might
                 handled = PlatformPeg.get().onKeyDown(ev);
         }
+
+        // Handle labs actions here, as they apply within the same scope
+        if (SdkConfig.get()['showLabsSettings'] && !handled) {
```

Comment:
I believe we should check for `SdkConfig.get()['showLabsSettings']` in `src/KeyBindingsDefaults.ts` instead

Reply:
`SdkConfig` is not initialized yet when that code runs.

Reply:
Really? From what I remember that would only run when a keybinding is actually pressed, though I might be wrong

Reply:
From my testing, `SdkConfig` hasn't loaded any configuration when `KeyBindingsDefaults` is ran, so for labs visibility just yields `undefined`. Should I create a separate file for labs keybinds, and use `registerShortcut()` within it?

Reply:
Yeah, looks like KeyBindingsDeafults is all just done at import-time. I think this is probably the correct place as far as I can see.

Reply:
Hmm, ok, I don't feel super happy about this we can figure out a way to improve upon this in the future

_Resolved_

Reply:
Ok, the check in `KeyBindingsDefaults` works, but it doesn't work in `KeyboardShortcuts`. I've moved the check there, and the shortcuts tab in settings checks whether labs shortcuts should be displayed at all.